CHANGELOG.md

```diff
@@ -116,6 +130,30 @@ New modules
   Data.Nat.PseudoRandom.LCG
   ```
 
+* Broke up `Data.List.Relation.Binary.Pointwise` and introduced:
+  ```
+  Data.List.Relation.Binary.Pointwise.Base
+  Data.List.Relation.Binary.Pointwise.Properties
+  ```
+
+* Broke up `Codata.Musical.Colist` into a multitude of modules:
+  ```
+  Codata.Musical.Colist.Base
+  Codata.Musical.Colist.Properties
+  Codata.Musical.Colist.Bisimilarity
```

Comment:
I know the `Musical` folder is morally deprecated, but should this be `Relation.Binary.Bisimilarity.Propositional`?

Reply:
I followed the structure we have in the sized `Codata` directories
where `Bisimilarity` is put front and center. Should we deprecate
these modules and put it all under `Codata.(...).Relation.Binary`?

Reply:
Ah yes. Hmm, probably I guess to be consistent. However, let's leave it for this PR. Already enough changes going on.